jaxrs-api/src/main/java/javax/ws/rs/JAXRS.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (c) 2018 Markus KARG. All rights reserved.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+
+package javax.ws.rs;
+
+import java.net.URI;
+import java.util.ServiceLoader;
+
+import javax.ws.rs.core.Application;
+
+/**
+ * Bootstrap class that is used to startup a JAX-RS application in Java SE
+ * environments.
+ *
+ * The {@code JAXRS} class is available in a Jakarta EE container environment as
+ * well; however, support for the Java SE bootstrapping APIs is <em>not
+ * required</em> in container environments.
+ *
+ * @author Markus KARG (markus@headcrashing.eu)
+ *
+ * @since 2.2
+ */
+interface JAXRS {
```

Comment:
Do we have a better name than JAXRS? A `Bootstrapper`, or an `ApplicationFactory` is more descriptive perhaps? Should this be an Interface? Abstract class could have `#bootstrap` method protected, if do not want to call it outside from `#start`.

I'd slightly prefer a `RuntimeDelegate` method such as `#createApplicationFactory()`, to avoid another static method.

Reply:
I like `JAXRS`. And I agree with @mkarg that it is consistent with `JAXB` and `Persistence`,

Reply:
I think current Java API trend is to only uppercase the first letter (see Jsonb). That would leave as with Jaxrs.

Don't have a preference. Just a thought to discuss.

Reply:
@ggam Good point.

Reply:
I cannot see a _trend_. This sounds more like personal taste.

Reply:
I don't love the use of the JAXRS name, but I understand the rationale for it. However, why isn't the implementation of the methods in this class not using RuntimeDelegate?